catlearn/ga/algorithm.py

```diff
@@ -17,7 +17,7 @@
 class GeneticAlgorithm(object):
     """Genetic algorithm for parameter optimization."""
 
-    def __init__(self, population_size, fit_func, features, targets,
+    def __init__(self, fit_func, features, targets, population_size=None,
```

Comment:
This might break some code. I wanted to put a default `nprocs=population_size * n`

Reply:
- Can we have this as `'auto'` instead of `None`? It is perhaps misleading currently.
- Can you then add some assertion with reasonable message that it should be `'auto'` or an `int`?